### ISSUE TITLE
Changing text

### ISSUE THREAD

**Issue by Brownjv14** [dotproject/dotProject#113]
Is there a way to change the form text's such as changing zip to postal code or state to province? thanks

**Comment by ajdonnison** [comment]
Yes, in most cases you can use the translation module.  You just need to define a local translation and you can translate pretty much the entire UI.  Look at those already in place for other languages.
